components/routing/src/Middleware/Authenticate.php

```diff
@@ -16,8 +16,8 @@ class Authenticate
      */
     public function handle($request, Closure $next, $guard = null)
     {
-        if (! $_SESSION['user']) {
-            return 'Error Authenticate. Please <a href="/login">login</a>';
+        if (isset($_SESSION['logged_in']) && ! $_SESSION['logged_in']) {
```

Comment:
Could you explain this change (logged_in with the isset and the check vs just user)? Thanks!